Misc/NEWS.d/next/Windows/2020-06-19-00-14-52.bpo-40851.0-3EJP.rst

```diff
@@ -0,0 +1,2 @@
+On Windows, :class:`subprocess.Popen` no longer sets ``wShowWindow`` if the
+``startupinfo`` argument has the flag ``STARTF_USESHOWWINDOW``.
```

Comment:
It describes what the code does, but not what it means for the user.

It can be acceptable in the commit message, but not in the NEWS entry. Please describe the change in terms of user visible effects.